diff --git a/dist/index.js b/dist/index.js
index bf539fca617f2b159e6d6bbc41c85330a66d21bc..be2936f099e81f9576d2d5050f3b0ed93f3f55f5 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,10 +1,10 @@
 // Copyright (C) myl7
 // SPDX-License-Identifier: Apache-2.0
 import { visit } from 'unist-util-visit';
-import { handleNode as handleNodeWithLegacyTitle } from './legacyTitle';
-import { nameFilter, classNameMap, defaultConfig, defaultConfigForLegacyTitle, } from './config';
+import { handleNode as handleNodeWithLegacyTitle } from './legacyTitle.js';
+import { nameFilter, classNameMap, defaultConfig, defaultConfigForLegacyTitle, } from './config.js';
 export { defaultConfig, defaultConfigForLegacyTitle };
-export { mkdocsConfig as mkdocsConfigForLegacyTitle } from './legacyTitle';
+export { mkdocsConfig as mkdocsConfigForLegacyTitle } from './legacyTitle.js';
 const plugin = function (providedConfig) {
     const legacyTitle = providedConfig?.legacyTitle ?? defaultConfig.legacyTitle;
     return (tree) => {
diff --git a/dist/legacyTitle.js b/dist/legacyTitle.js
index 48c2dd19d6338dcb1272157bad57b686fea48d7c..e9657cf444661d1879a35ab2ed28f577c35a68c4 100644
--- a/dist/legacyTitle.js
+++ b/dist/legacyTitle.js
@@ -1,6 +1,6 @@
 // Copyright (C) myl7
 // SPDX-License-Identifier: Apache-2.0
-import { classNameMap, nameFilter } from './config';
+import { classNameMap, nameFilter } from './config.js';
 export const handleNode = (config) => (node) => {
     // Filter required elems
     if (node.type != 'blockquote')